apps/server/src/modules/provisioning/strategy/oidc/service/oidc-provisioning.service.spec.ts

```diff
@@ -11,8 +11,11 @@ import { RoleDto } from '@src/modules/role/service/dto/role.dto';
 import { SchoolService } from '@src/modules/school';
 import { UserService } from '@src/modules/user';
 import CryptoJS from 'crypto-js';
+import { federalStateFactory, schoolDOFactory } from '@shared/testing';
+import { schoolYearFactory } from '@shared/testing/factory/schoolyear.factory';
 import { ExternalSchoolDto, ExternalUserDto } from '../../../dto';
 import { OidcProvisioningService } from './oidc-provisioning.service';
+import { FederalStateService, SchoolYearService } from '../../../../school/service';
```

Comment:
Can you please export it over school/index and import it over @src/modules/school
The folder structure deeper then school/ should be unknown from the view of this modul.